assets/styles/icons.scss

```diff
@@ -11,150 +11,150 @@
 
 .icon-letter:before {
     @extend %icon;
-    font-icon: 150% url('letter.svg');
+    font-icon: 150% url('../../assets/icons/letter.svg');
```

Comment:
If `postcss-fonticons` works correctly, this shouldn't be necessary.

Reply:
Way ahead of you. :P